src/api/init.ts

```diff
@@ -155,10 +155,20 @@ export async function startApiServer(opts: {
       format: logger.format,
       transports: logger.transports,
       metaField: (null as unknown) as string,
-      statusLevels: {
-        error: 'error',
-        warn: httpLogLevel ?? 'http',
-        success: httpLogLevel ?? 'http',
+      statusLevels: false,
```

Comment:
If you set this to `true`, I believe it will essentially implement the same logic as below [according to the docs](https://www.npmjs.com/package/express-winston):
> If you set statusLevels to true express-winston will log sub 400 responses at info level, sub 500 responses as warnings and 500+ responses as errors.

Reply:
Interesting... this passed by my side. Way simpler than customizing every response code. 

👍